src/components/ui/input/input.tsx

```diff
@@ -44,9 +46,9 @@ export const Input = ({
         </label>
       )}
       <input
-        className={`${error != null ? "border-red-500" : ""} border px-4 py-1.5 rounded-md ${
-          isMobile ? "text-sm" : "text-base"
-        }`}
+        className={`w-${width}${
```

Comment:
remove `w-` here and add space between `${width}` and `${`

Reply:
done :3